examples/src/Examples.hpp

```diff
@@ -1645,3 +1645,96 @@ struct ex_split_merge : public labsound_example
     }
     
 };
+// https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createWaveShaper#example
+
+static void makeDistortionCurveA(std::vector<float> & curve, unsigned n_samples, float amount = .5f)
+{
+    float k = amount * 10.0f;
+    float deg = (float) std::_Pi / 180.0f;
```

Comment:
is std::_Pi MSVC specific? Maybe we are stuck with M_PI here for cross platform-ness?